makefile

```diff
@@ -187,7 +187,8 @@ DEBUG_SET_ENV_VARS := \
 	export FEATURE_EU_EXIT_FORMS_ENABLED=true; \
 	export EU_EXIT_ZENDESK_SUBDOMAIN=debug; \
 	export FEATURE_CONTACT_US_ENABLED=true; \
-	export EUEXIT_AGENT_EMAIL=test@example.com
+	export EUEXIT_AGENT_EMAIL=test@example.com; \
+	export EUEXIT_GOV_NOTIFY_TEMPLATE_ID=debug
```

Comment:
this will break submissions of euexit forms on local env.

please put `EUEXIT_GOV_NOTIFY_TEMPLATE_ID=debug` in your `conf/.env` file :)

Reply:
whoops. left this in